### ISSUE TITLE
[Project]: Arrangement of project directory

### ISSUE THREAD

**Issue by smilewithkhushi** [smilewithkhushi/BasicNative#111]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

**Under this issue, you need to work on improving the directory structure of this project.**

Currently, a few screens have been stored in the main directory, that is : 

![image](https://github.com/user-attachments/assets/39ad35a2-d58d-485d-a4a5-b16e3f4c3913)

There is a need to move all these mini-projects folders, namely My chat app, sos app, weather app,seating chart, video calling app to the dedicated directory.

The destination directory maps as follows :  src/screens/*Your project name*/*your project files*

![image](https://github.com/user-attachments/assets/0495dc57-d8b7-4f46-b0fd-b4eac40299e1)

In short, you have to : 
1. Move all these mini projects to the destination directory "src/screens/"
2. Update the linking of these mini projects on the landing screen.
3. Report if you find any bugs within the project and raise an issue!

### Use Case

This feature involves the management of the project directory

### Benefits

This will make it easier for the future contributors  and developers to understand the project.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [] I'm a GSSOC'24 contributor
- [] I want to work on this issue

**Comment by Sawan-Kushwah** [comment]
can i work on it please assign me this

**Comment by menerucha** [comment]
I can work on this issue please assign me under gssoc-ext

**Comment by smilewithkhushi** [comment]
@Sawan-Kushwah assigned! get this done in 2 days and raise a PR.
all the best!

**Comment by Sawan-Kushwah** [comment]
![image](https://github.com/user-attachments/assets/f6d2f756-62d0-43cc-ba81-23fe41ba1461)

@smilewithkhushi will you please help in setting up the project?

**Comment by menerucha** [comment]
Yes sure, I will do it within 2 days
